### PR TITLE
Add Grouped Option Group story

### PR DESCRIPTION
**Pull request by azizhk**

### CODE REVIEW

**Comment by ritz078** [comment]
This can be achieved by just creating a `SectionTitle` component. Since iteration is done by the user, they can put the `SectionTitle` component when they need.

**Comment by azizhk** [comment]
This is now ready to review and merge.

File: src/components/__tests__/phonenumberinput.test.tsx [hunk 1]
```diff
@@ -88,30 +88,30 @@ describe("Component: Select", () => {
...
+    //     value: ""
+    //   }
+    // });
+
+    // expect(
+    //   spy.calledWith({
+    //     phone: "",
+    //     countryCode: "+1"
+    //   })
+    // ).toBeTruthy();
```
Comment (ritz078) [inline src/components/__tests__/phonenumberinput.test.tsx]: why commented?

**Comment by ritz078** [comment]
Later we should do this https://github.com/ritz078/pebble/issues/158

File: src/components/__tests__/phonenumberinput.test.tsx (hunk 1 above)
Comment (azizhk) [reply src/components/__tests__/phonenumberinput.test.tsx]: Because these tests were actually wrong.
Fixing them.

**Comment by ritz078** [comment]
needs `userSelect: "none"` in CSS
screenshot 2018-12-13 at 5 25 30 pm

**Pull request merged**